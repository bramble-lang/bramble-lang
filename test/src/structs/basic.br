fn my_main() {
    let x:i32 := 1;
    printiln x;
    return;
}

struct MyStruct {
    x: i32,
    y: bool,
}

struct MyStruct2 {}

fn test(ms: MyStruct) -> MyStruct {
    let t: MyStruct := MyStruct{x: 5, y: true};
    return ms;
}