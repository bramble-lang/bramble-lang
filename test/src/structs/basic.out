5
1
true
5
5
